src/core/data/ipls.hpp

```diff
@@ -5,18 +5,79 @@ struct ipl_libary
+        {{""}},
+        {{""}},
+    }, // Cayo Perico Dont Load Interiors Why(?
+    {
+        "Dignity Party Yacht", 
+        {-2045.8, -1031.2, 11.9}, 
+        {{"smboat"}, {"apa_smboat_lodlights"}, {"ba_sm_boat_window"}},
+        {{""}},
+        {{""}},
+    }, // Dignity Party Yacht
+    {
+        "Aircraft Carrier",
+        {3069.33, -4632.4, 15.04},
+        {{"hei_carrier_lodlights"}, {"hei_carrier"}, {"hei_carrier_int1"}, {"hei_carrier_int2"}, {"hei_carrier_int3"}, {"hei_carrier_int4"}, {"hei_carrier_int5"}, {"hei_carrier_int6"}},
+        {{""}},
+        {{""}},
+    }, // Aircraft Carrier
+    {
+        "Sunken Cargo Ship",
+        {-162.89, -2365.76, 0.0},
+        {{"sunk_ship_fire"}, {"sunkcargoship"}},
+        {{"cargoship"}},
+        {{"cargoship"}},
+    }, // Sunken Cargo Ship (Merryweather heist)
+    {
+        "Destroyed Pillbox Hospital", 
+        {297.84, -584.41, 43.26}, 
+        {{"rc12b_hospitalinterior"}},
+        {{"rc12b_default"}},
+        {{"rc12b_default"}},
+    }, // Destroyed Pillbox Hospital
+    {
+        "O'Neil Farm Destroyed",
+        {2471.49, 4954.18, 45.12},              
+        {{"farm_burnt"}, {"farm_burnt_props"}}, 
+        {{"farm"}, {"farmint"}, {"farmint_cap"}, {"farm_props"}, {"des_farmhs_startimap"}}, 
+        {{"farm"}, {"farmint"}, {"farmint_cap"}, {"farm_props"}, {"des_farmhs_startimap"}}, 
+    }, // Farm Destroyed House
+    {
+        "LifeInvader Interior",
+        {-1082.77, -258.67, 37.76}, 
+        {{"facelobby"}},
+        {{"facelobbyfake"}},
+        {{"facelobbyfake"}},
+    }, // LifeInvader Interior
+    {
+        "Jewelry Store Interior", 
+        {-630.4, -236.7, 40.0},
+        {{"post_hiest_unload"}},
+        {{"jewel2fake"}},
+        {{"jewel2fake"}, {"bh1_16_refurb"}},
+    }, // Jewelry Interior
+    {"Coroner Morgue Interior", 
+    {244.9, -1374.7, 39.5},
+    {{"coroner_int_on"}, {"coronertrash"}},
+    {{"coroner_int_off"}},
+    {{"coroner_int_off"}},
+    }, // Coroner Morgue Interior
```

Comment:
There's a lot more IPLs you could load, would be actually useful like the Arcade or Casino, but that's an issue for a different PR.

Reply:
I will add all the ipls, in a later pr c: